ratelimit/exceptions/__init__.py

```diff
@@ -0,0 +1,3 @@
+# flake8: noqa: F401
+from .backend_connection import BackendConnectionException
```

Comment:
I think we should put the exceptions in a file, it seems unnecessary to break them down for our codebase.

Reply:
Sure, I moved them to a single file `exceptions.py`